homeassistant/helpers/llm.py

```diff
@@ -434,6 +447,10 @@ def _get_exposed_entities(
         "current_temperature",
         "temperature_unit",
         "brightness",
+        "color_temp_kelvin",
+        "rgb_color",
+        "color_mode",
+        "supported_color_modes",
```

Comment:
I don't know about these values, it's adding a lot of extra data.

Reply:
We could possibly drop `color_temp_kelvin` and `supported_color_modes` and keep `rgb_color`. As for `supported_color_modes`, I'll have to see what happens. It may break prompts that interact with groups of lights that have mixed color mode support.

Reply:
I was discussing this with the LLM group, and we felt that more attributes should probably be hidden behind a tool call to get more info on specific states. That way the base prompt can be smaller, and we can include more info when needed.

Reply:
Feel free to close in that case.

If you wanna keep just the color, I wonder if we can compress the current RGB color into a packed three byte hex to save on space (e.g. add a brief sentence describing the color format and then turn `#aabbccdd` -> `abc`).